katib/katib-controller/overlays/application/application.yaml

```diff
@@ -6,11 +6,17 @@ spec:
   addOwnerRef: true
   componentKinds:
     - group: core
-      kind: Service
+      kind: ConfigMap
```

Comment:
Do we need to also add other components from base: https://github.com/kubeflow/manifests/tree/master/katib/katib-controller/base?
ClusterRole, ClusterRoleBinding and PersistentVolumeClaim ?

Reply:
@andreyvelich you can NOT put cluster level resources in the Application selector, as applications are Namespaced resources, everything would break.

Re: PersistentVolumeClaim, it depends on if we want to allow users to accidentally delete their PVCs by deleting their Application.

Reply:
In that case, I think we can skip them.